Xiaomi/Redmi7A/res/values/config.xml

```diff
@@ -0,0 +1,95 @@
+  </integer-array>
+
+  <integer-array name="config_autoBrightnessLevels">
+      <item>1</item>
+      <item>2</item>
+      <item>4</item>
+      <item>5</item>
+      <item>8</item>
+      <item>13</item>
+      <item>17</item>
+      <item>21</item>
+      <item>26</item>
+      <item>30</item>
+      <item>34</item>
+      <item>39</item>
+      <item>60</item>
+      <item>140</item>
+      <item>310</item>
+      <item>400</item>
+      <item>500</item>
+      <item>600</item>
+      <item>1000</item>
+      <item>1200</item>
+      <item>1500</item>
+      <item>3000</item>
+      <item>3500</item>
+      <item>4000</item>
+  </integer-array>
+
+    <bool name="config_automatic_brightness_available">true</bool>
+    <bool name="config_setColorTransformAccelerated">true</bool>
+    <bool name="config_supportAudioSourceUnprocessed">true</bool>
+    <bool name="config_useDevInputEventForAudioJack">true</bool>
+    <bool name="skip_restoring_network_selection">true</bool>
+    <bool name="config_bluetooth_le_peripheral_mode_supported">true</bool>
+    <bool name="config_carrier_volte_available">false</bool>
+    <bool name="config_device_volte_available">false</bool>
+    <bool name="config_device_vt_available">false</bool>
+    <bool name="config_device_wfc_ims_available">false</bool>
+    <bool name="config_hotswapCapable">true</bool>
+    <bool name="config_lidControlsSleep">true</bool>
+    <bool name="config_wifiDisplaySupportsProtectedBuffers">true</bool>
+    <bool name="config_wifi_background_scan_support">true</bool>
+    <bool name="config_wifi_batched_scan_supported">true</bool>
+    <bool name="config_wifi_dual_band_support">false</bool>
+    <bool name="config_dozeAlwaysOnDisplayAvailable">false</bool>
+    <bool name="config_displayBlanksAfterDoze">true</bool>
+    <bool name="config_powerDecoupleAutoSuspendModeFromDisplay">true</bool>
+    <fraction name="config_autoBrightnessAdjustmentMaxGamma">100.0%</fraction>
+    <fraction name="config_maximumScreenDimRatio">29.999996%</fraction>
+    <integer name="config_autoBrightnessBrighteningLightDebounce">2000</integer>
+    <integer name="config_autoBrightnessDarkeningLightDebounce">2000</integer>
+    <integer name="config_screenBrightnessDark">1</integer>
+    <integer name="config_screenBrightnessDim">1</integer>
+    <integer name="config_screenBrightnessDoze">17</integer>
+    <integer name="config_screenBrightnessSettingDefault">536</integer>
+    <integer name="config_screenBrightnessSettingMaximum">2048</integer>
+    <integer name="config_screenBrightnessSettingMinimum">7</integer>
+    <integer name="config_bluetooth_idle_cur_ma">0</integer>
+    <integer name="config_bluetooth_operating_voltage_mv">3300</integer>
+    <integer name="config_bluetooth_rx_cur_ma">0</integer>
+    <integer name="config_bluetooth_tx_cur_ma">0</integer>
+    <integer name="config_shutdownBatteryTemperature">600</integer>
+    <dimen name="rounded_corner_radius">0px</dimen>
```

Comment:
if i'm not mistaken, this doesn't work, does it?
I seem to remember this must be in systemui not framework-res

Reply:
nah, i think i checked most of the values to be in framework res, and edited them to match 7A's values